CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Testing locally
+
+To learn how to run the tests from Github actions locally in containers, see
+[`containers`](https://github.com/OpenSC/OpenSC/master/containers/).
```

Comment:
Github [supports relative links](https://github.blog/2013-01-31-relative-links-in-markup-files/); should look a bit nicer.

Reply:
ok. Changed to relative.